pkgs/applications/editors/neovim/utils.nix

```diff
@@ -236,5 +237,6 @@ in
   inherit normalizePlugins normalizedPluginsToVimPackage;
 
   inherit buildNeovimPlugin;
+} // lib.optionalAttrs config.allowAliases {
```

Comment:
fine by me. How did you find it  ? I intend to remove it altogether after 24.11 if no one objects.

Reply:
I wrote a nix script which generates a JSON file with metadata about packages. These were some of the trace warnings that I got while running it.

https://gist.github.com/jopejoe1/6eba2d6f628e9897219ce64484434c14